index.d.ts

```diff
@@ -0,0 +1,6 @@
+declare module 'should-send-same-site-none' {
+  import { RequestHandler } from 'express';
```

Comment:
@tomselvi I hit an issue with this `import` since I'm not using `express` at all. Getting this when compiling;

```
$ tsc

node_modules/should-send-same-site-none/index.d.ts:2:34 - error TS2307: Cannot find module 'express'.
```
I'd still really want to use this package though. You know of any tricks I could use to get past this TypeScript error?

Reply:
@phillipj - are you familiar with adding a typescript module declaration? You could add one and then include it in your tsconfig.json?  Almost like a stub for express but then you could just use the express-less function with no issues.  I can put together a little side demo for you if that helps

Reply:
Thanks for replying @tomselvi!

Yes, to make the compiler happy, I copy-pasted the definitions you've added here and removed the `import` referencing express.

```ts
// ./types/should-send-same-site-none.d.ts

declare module "should-send-same-site-none" {
  export function isSameSiteNoneCompatible(useragent: string): boolean;
}
```

And added it to the `tsconfig.json` (shortened for brevity):

```json
{
  "compilerOptions": {
    "paths": {
      "should-send-same-site-none": ["types/should-send-same-site-none.d.ts"]
    }
  },
  "include": ["src/**/*"],
}
```

Sounds like what you suggested, thanks for confirming that's a viable approach 😃